### PR TITLE
Pull to refresh

### PR DESCRIPTION
**Pull request by argelius**
@anatoo さん、

ons-pull-hookを実装しました。もし何か間違いがあれば言ってください。
ドキュメントの翻訳もお願いします。

### CODE REVIEW

File: framework/views/pullHook.js [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+
+        this._translateTo(0, { animate: true });
+        this._setState(this.STATE_INITIAL);
+      },
+
+      _safeApply: function() {
+        var phase = this._scope.$root.$$phase;
+
+        if (phase !== '$apply' && phase !== '$digest') {
+          this._scope.$apply();
```
Comment (anatoo) [inline framework/views/pullHook.js]: Request: Don't refer $$phase to implement safeApply(). That is not recommended on AngularJS wiki(https://github.com/angular/angular.js/wiki/When-to-use-%24scope.%24apply%28%29). Use $timeout() or $scope.$evalAsync().

File: framework/views/pullHook.js [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+        }
+      },
+
+      _getHeight: function() {
+        return parseInt(this._element[0].getAttribute('height') || '64');
```
Comment (anatoo) [inline framework/views/pullHook.js]: Request: Pass second parameter on parseInt() function. On old browsers, parseInt('010') return 8.

File: framework/directives/pullHook.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      restrict: 'E',
+      replace: false,
+      scope: true,
+      compile: function(element, attrs) {
+        return {
+          pre: function(scope, element, attrs) {
+            var pullHook = new PullHookView(scope, element, attrs);
+
+            $onsen.declareVarAttribute(attrs, pullHook);
+            $onsen.aliasStack.register('ons.pullHook', pullHook);
```
Comment (anatoo) [inline framework/directives/pullHook.js]: Request: Remove this line.

**Comment by argelius** [comment]
@anatoo, thank you for looking at the code and making the translations.

I changed to an `$evalAsync'  call instead and it works well.

**Comment by argelius** [comment]
Merged.

**Pull request merged**